joft/utils.py

```diff
@@ -15,6 +15,22 @@ def load_and_parse_yaml_file(path: str) -> typing.Dict[str, typing.Any]:
     return yaml_obj
 
 
+def read_and_validate_config(
+    path: str | pathlib.Path,
+) -> typing.Dict[str, typing.Any] | None:
+    """Read and return config file is it's valid. Return None otherwise."""
+    with open(path, "rb") as fp:
+        config = tomllib.load(fp)
+
+    try:
+        config["jira"]["server"]["hostname"]
+        config["jira"]["server"]["pat_token"]
+    except KeyError:
```

Comment:
This function now raises on IO error, TOML errors, but returns None one KeyErrors. I find that inconsistent.

If you raise the KeyError, it has additional information that can be presented to the user.